src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,122 @@
+const expensesService = require('../services/expenses.service');
+const userService = require('../services/user.service');
+
+const get = async (req, res) => {
+  const { userId, from, to, categories } = req.query;
+  const expenses = await expensesService.getAllExpenses();
```

Comment:
Consider adding error handling for the async call to expensesService.getAllExpenses(). If an error occurs, it should be caught and an appropriate response should be sent to the client.